apps/docs/pages/guides/platform/custom-domains.mdx

```diff
@@ -18,139 +18,213 @@ Custom domains allow you to present a branded experience to your users. Custom d
   ></iframe>
 </div>
 
-## Configure a Custom Domain using the Supabase Dashboard
+There are two types of customizable domains supported by Supabase:
+
+1. Custom domains, where you use a domain such as `api.example.com` instead of the project's default domain.
+2. Vanity subdomains (experimental), where you can setup a different Subdomain on `supabase.co` for your project.
+
+You can only configure one or the other at this time.
+
+## Custom domains
+
+Custom domains allow you to change the way your project's URLs appear to your users. This is useful when:
+
+- You are using [OAuth (Social Login)](/docs/guides/auth/social-login) with Supabase Auth and the project's URL is shown on the OAuth consent screen.
+- You are creating APIs for third-party systems, like when implementing webhooks or external API calls to your project via [Edge Functions](/docs/guides/functions).
+- You are storing URLs in a database or encoding them in QR codes.
+
+Configuring a custom domain also helps you keep your APIs portable for the long-term. By using a custom domain you can easily migrate from one Supabase project to another, or make it easier to version APIs in the future.
+
+### Configure a custom domain using the Supabase dashboard
 
 Follow the **Custom Domains** steps in the [General Settings](https://supabase.com/dashboard/project/_/settings/general) page in the Dashboard to set up a custom domain for your project.
 
-## Configure a Custom Domain using the Supabase CLI
+### Configure a custom domain using the Supabase CLI
 
-This example assumes your Supabase project is `foobarbaz` with a corresponding API URL `foobarbaz.supabase.co` and configures a custom domain at `api.example.com`.
+This example assumes your Supabase project is `abcdefghijklmnopqrst` with a corresponding API URL `abcdefghijklmnopqrst.supabase.co` and configures a custom domain at `api.example.com`.
 
 To get started:
 
-1. [Install](/docs/guides/resources/supabase-cli) the Supabase CLI 1.11.3+.
-1. [Log in](/docs/guides/resources/supabase-getting-started/local-development#log-in-to-the-supabase-cli) to your Supabase account using the CLI.
+1. [Install](/docs/guides/resources/supabase-cli) the latest version of the Supabase CLI.
+2. [Log in](/docs/guides/resources/supabase-getting-started/local-development#log-in-to-the-supabase-cli) to your Supabase account using the CLI.
+3. Setting up a custom domain requires [Owner or Admin permissions](/docs/guides/platform/access-control#manage-team-members) for the project.
+4. Get a custom domain. Currently, only subdomains are supported.
```

Comment:
```suggestion
4. Get a custom domain from a DNS provider. Currently, only subdomains are supported.
```

flexible on the text, just want to make it clear this part has to be done with someone else and isn't something you configure on our dashboard